### PR TITLE
docs: Adds administrator manual

### PR DESCRIPTION
**Pull request by vikram-dagger**
This commit adds a new top-level category for the "administrator manual". This is a parallel category to the "user manual" and "developer manual" being added in #6869 

This commit:
- transfers the CI code samples from the "user manual -> CI" section to a new "integrations" section
  - it also adds pages for Tekton and Kubernetes integrations based on the guides at https://archive.docs.dagger.io/0.9/194031/kubernetes, https://archive.docs.dagger.io/0.9/237420/ci-architecture-kubernetes and https://archive.docs.dagger.io/0.9/213240/tekton
  - redirections are in place for the changes; however note that this will impact some of the links on https://dagger.io/integrations @mircubed @d3rp3tt3 
- transfers administration-related pages from the "user manual -> Dagger Cloud" section to an "admin manual -> Dagger Cloud" section 
   - redirections are in place for the changes; however note that this will impact the docs link in https://dagger.cloud @d3rp3tt3 
- transfers content related to setting up a custom runner/custom registry mirror from the [operator guide](https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md) to an "admin manual -> Dagger Engine" section 
- ~~adds a new "admin manual -> tutorials" section for administrator-related tutorials~~
  - ~~it populates this "tutorials" section with a slightly-revised version of the content at~~
    -  ~~https://archive.docs.dagger.io/0.9/934191/eks-github-karpenter~~
    - ~~https://archive.docs.dagger.io/0.9/488564/openshift-gitlab~~
    - ~~https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md~~

### CODE REVIEW

File: docs/current_docs/manuals/administrator/cloud/609601-organizations.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/cloud/609601-organizations.mdx:2]: Ping @d3rp3tt3 to note the above change

File: docs/current_docs/manuals/administrator/cloud/968372-roles-permissions.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/cloud/968372-roles-permissions.mdx:2]: Ping @d3rp3tt3 to note the above change

File: docs/current_docs/manuals/administrator/integrations/728101-tekton.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/integrations/728101-tekton.mdx:2]: @gerhard @kpenfound can you please review this page? I have based it on https://archive.docs.dagger.io/0.9/213240/tekton, modified to use a function call. However, I was not able to test it.

File: docs/current_docs/tutorials/934191-eks-github-karpenter.mdx
Comment (vikram-dagger) [inline docs/current_docs/tutorials/934191-eks-github-karpenter.mdx:88]: This section has changed from the previous version of the guide. Instead of using a sample Node app and a custom SDK-based script, I have updated it to use Kyle's Go builder module and the Dagger repo. Let me know if this is fine?

File: docs/current_docs/manuals/administrator/integrations/index.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/integrations/index.mdx:2]: @d3rp3tt3 I'm not sure if we are referring to specific pages from the Dagger Cloud UI. If we are, then please note these URL changes (redirections will be in place though). Inline links within the Dagger Cloud get started guide will be updated separately (not in this PR).

File: docs/netlify.toml
Comment (vikram-dagger) [inline docs/netlify.toml:42]: FYI @d3rp3tt3

**Review by vikram-dagger — COMMENTED** [review]
@kpenfound @gerhard @d3rp3tt3 @sipsma please review

**Comment by vikram-dagger** [comment]
@shykes please review and let me know your thoughts. A few questions:
- Should "integrations" be moved up on level ie. to the top level of the nav tree?
- Should the list of integration be further categorized?

File: docs/current_docs/manuals/administrator/integrations/248292-openshift.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/integrations/248292-openshift.mdx:2]: @gerhard @kpenfound can you please review this page? I have based it on https://archive.docs.dagger.io/0.9/488564/openshift-gitlab, modified to use a function call.

File: docs/current_docs/tutorials/488565-openshift-gitlab.mdx
Comment (vikram-dagger) [inline docs/current_docs/tutorials/488565-openshift-gitlab.mdx:9]: @gerhard @kpenfound this is a straight copy of https://archive.docs.dagger.io/0.9/488564/openshift-gitlab with minor changes. Please let me know if anything stands out as needing changes.

File: docs/netlify.toml
Comment (vikram-dagger) [inline docs/netlify.toml:62]: @d3rp3tt3 @mircubed this will affect some of the links on https://dagger.io/integrations. Redirections are in place in this PR but it would better to update the links anyway to point to the new pages.

File: docs/current_docs/tutorials/272381-custom-registry.mdx
Comment (vikram-dagger) [inline docs/current_docs/tutorials/272381-custom-registry.mdx:9]: @sipsma this is derived from https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md. Could you please review and let me know if any changes are needed?

File: docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx
Comment (vikram-dagger) [inline docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx:5]: @sipsma this is derived from https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md. Could you please review and let me know if any changes are needed?

File: docs/docusaurus.config.ts
Comment (vikram-dagger) [inline docs/docusaurus.config.ts:64]: Note: if this PR is merged before #6869, some tutorials will be hidden as they will be in a different location.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (shykes) [inline docs/current_docs/manuals/administrator/tutorials/index.mdx:10]:
```suggestion
```

This can be a top-level section in the Administrator manual.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (shykes) [inline docs/current_docs/manuals/administrator/tutorials/index.mdx:11]:
```suggestion
```

This should be merged into the Gitlab CI integration page, as a special optimization ("Self-hosted CI runner + Kubernetes Daemonset")

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (shykes) [inline docs/current_docs/manuals/administrator/tutorials/index.mdx:12]:
```suggestion
```

This should be merged into 1) the Github Actions integration page, as a special optimization, and 2) a new EKS integration page.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (shykes) [inline docs/current_docs/manuals/administrator/tutorials/index.mdx:9]:
```suggestion
```

The tutorials section can be removed - see other comments for what to do with each of the 3 articles.

**Review by shykes — CHANGES_REQUESTED** [review]
As discussed live today with @vikram-dagger.

Note: I recommend breaking this up in 2 PRs:

1. Merge new admin manual *without tutorials section*. This can be fast-tracked.
2. Add contents from the tutorial section, merged into integrations content. This can happen later.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/tutorials/index.mdx]: Done

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/tutorials/index.mdx]: OK. I have removed this from the current PR and noted this in Linear for follow-up in a separate PR.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/tutorials/index.mdx]: OK. I have removed this from the current PR and noted this in Linear for follow-up in a separate PR.

File: docs/current_docs/manuals/administrator/tutorials/index.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/tutorials/index.mdx]: Done

**Comment by vikram-dagger** [comment]
> As discussed live today with @vikram-dagger.
> 
> Note: I recommend breaking this up in 2 PRs:
> 
> 1. Merge new admin manual _without tutorials section_. This can be fast-tracked.
> 2. Add contents from the tutorial section, merged into integrations content. This can happen later.

OK. I have removed the tutorials from the current PR and noted the pending items in Linear for follow-up in a separate PR.

**Comment by vikram-dagger** [comment]
@shykes please re-review - thanks!

**Review by shykes — APPROVED** [review]
🚀🚀🚀

**Comment by gerhard** [comment]
Doing a quick pass now - OK to merge before if you want to get this out @vikram-dagger.

File: docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx
Comment (gerhard) [inline docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx:35]: I would update this to a newer tag, e.g. `v0.10.2`

File: docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx
Comment (gerhard) [inline docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx:41]: This issue was closed in favour of the addition that we added to the internal operator manual in:
- https://github.com/dagger/dagger/pull/5809

**Review by gerhard — APPROVED** [review]
I've left a few tiny comments - no blockers from my side, OK to merge & publish 💪

This is a great start!
image

File: docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx]: Done

File: docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx
Comment (vikram-dagger) [reply docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx]: Deleted sentence and link

**Comment by vikram-dagger** [comment]
> Doing a quick pass now - OK to merge before if you want to get this out @vikram-dagger.

If possible, I'm going to wait to merge this until #6869 is also approved, because there are some interdependencies between the two PRs. eg. the Integrations section in this manual overrides the CI section in the user manual in #6869 and eg. language consistency, because it will be odd to see a user "guide" and an admin "manual" coexisting in the nav bar.

File: docs/current_docs/manuals/administrator/engine/272381-custom-registry.mdx
Comment (sipsma) [inline docs/current_docs/manuals/administrator/engine/272381-custom-registry.mdx:7]:
```suggestion
The Dagger Engine can be configured to use container registry mirrors for any registry URL. This allows container images that refer to one registry to instead be redirected to a different one.
```

Just want to clarify that there isn't a default; this feature lets you setup a mirror for any registry url.

**Review by sipsma — APPROVED** [review]
One tiny comment, otherwise the `administrator/engine` files LGTM!

**Pull request merged**
